### PR TITLE
Addapt getHtmlForAddColumn to use twig template

### PR DESCRIPTION
**Pull request by laps15**
I started to adapt some functions that returns html string and are not yet using Twig templates.

Signed-off-by: Leonardo Strozzi <laps15@inf.ufpr.br>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: libraries/classes/CentralColumns.php [hunk 1]
```diff
@@ -875,39 +875,25 @@ public function getHtmlForColumnDropdown($db, $selected_tbl)
...
+     public function getHtmlForAddColumn(
+         $total_rows,
+         $pos,
+         $db
+     ) {
+         $icon = Util::getIcon(
+             'centralColumns_add',
+             __('Add column')
+         );
+         $hidden_inputs = Url::getHiddenInputs($db);
```
Comment (mauriciofauth) [inline libraries/classes/CentralColumns.php]: You can replace these methods with their respective Twig extensions.
For example: `{{ Url_getHiddenInputs(db) }}`

File: templates/database/central_columns/add_column.twig [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <td class="central_columns_navigation">
+      {{ icon|raw }}
+      <form id="add_column" action="db_central_columns.php" method="post">
+        {{ hidden_inputs|raw }}
+        <input type="hidden" name="add_column" value="add">
+        <input type="hidden" name="pos" value="{{ pos }}" />
+        <input type="hidden" name="total_rows" value="{{ total_rows }}"/>
+        {{ table_drop_down|raw }}
+        <select name="column-select" id="column-select">
+          <option value="" selected="selected">Select a column.</option>
```
Comment (mauriciofauth) [inline templates/database/central_columns/add_column.twig]: You must make this string translatable. In that case you can use the `trans` tag.
For example: `{% trans 'Select a column.' %}`

File: templates/database/central_columns/add_column.twig [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        </select>
+      </form>
+    </td>
+    <td class="navigation_separator largescreenonly"></td>
+  </tr>
```
Comment (mauriciofauth) [inline templates/database/central_columns/add_column.twig]: We are using four spaces for the indentation of the twig files.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
You need to change the test for this method to reflect the changes.

See https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/356166110#L700

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**